server/utils/summaryList.ts

```diff
@@ -3,9 +3,16 @@ export enum ListStyle {
   bulleted,
 }
 
+interface Caseworker {
```

Comment:
this seems like a strange place to define this interface. 

do we need it at all? can we just use `AuthUserDetails`?

Reply:
yeah I had this originally, but felt that it was better to have something less specific and less coupled. 
I've replaced Caseworker with AuthUserDetails.